src/libcollections/btree/node.rs

```diff
@@ -200,6 +200,9 @@ struct RawItems<T> {
     tail: *const T,
 }
 
+unsafe impl<T: Send> Send for RawItems<T> { }
```

Comment:
Good catch. I assumed `RawItems` is safe since it has `Iterator` and `DoubleEndedIterator` implementations which take `&mut self`. But by closer inspection, the real mutations are all in `unsafe` block. `MoveTraversalImpl` is also a very good point.

Comment addressed.